tutorcairn/templates/cairn/apps/openedx/scripts/server.py

```diff
@@ -0,0 +1,45 @@
+from aiohttp import web
```

Comment:
- nit: a file docstring will be gelpful to understand the context.
- Please check that imports are ordered correctly (first party \n third party \n local code)

Reply:
I've sorted the imports and added a file docstring as well.